runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -193,11 +194,11 @@ private void rebuildAsync(boolean updated, boolean skillPaused, XpSnapshotSingle
 			}
 
 			paused = skillPaused;
-
```

Comment:
Probably don't remove this line since it separates the state updating from the label building